lib/fluent/config/section.rb

```diff
@@ -106,7 +106,7 @@ def method_missing(name, *args)
     end
 
     module SectionGenerator
-      def self.generate(proxy, conf, logger, plugin_class, stack = [])
+      def self.generate(proxy, conf, logger, plugin_class, strict_config_value = false, stack = [])
```

Comment:
Because I checked all callers in fluentd then I found that there is a case that `stack` is omitted and `strict_config_value` should be specified. The reversed case doesn't seem exist.
But I've notice that I didn't care the case that plugins may use it. I'll address it.

Reply:
done.